changelog/seqera-cloud/25.1.0_cycle5.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Seqera Cloud v25.1.0_cycle5
+date: 2025-03-19
+tags: [seqera cloud]
+---
+
+:::note
+From Wednesday, February 26, 2025, Seqera endpoints no longer support Windows XP and Safari versions 8 and below (including OSx 10.10 or older). This change is necessary to ensure the security and performance of our services. We recommend updating your operating system and browser version to ensure uninterrupted access to our services.
+
+Affected endpoints: 
+- cr.seqera.io
+- *.connect.cloud.seqera.io
+- intern.seqera.io
+- ai.seqera.io
+- hub.seqera.io
+- cloud.seqera.io
+- api.cloud.seqera.io
+- user-data.cloud.seqera.io
+- tower.nf
+- api.tower.nf
+- public.cr.seqera.io
+- auth.cr.seqera.io
+- wave.seqera.io
+- licenses.seqera.io
+- licman.seqera.io
+- api.multiqc.info
+:::
+
+### Feature updates and improvements
+
+- **Studios**
+  - Private sessions: When adding a new session, the number of concurrent connections can now be defined in the **General Config > Collaboration**.
+  - Dynamic studio templates: Labels have been added to indicate the status of support (e.g., *Recommended*, *Deprecated*) for a Studio template version. Users can now also switch the Studio's base template image when starting a session as new.
+- **Seqera Compute**
+  - Updated Seqera Compute compute environment creation form with new fields: Advanced options, Pre/post run scripts, Nextflow config.
+  - Added a **Features** tab to the admin workspace management page and a toggle with confirmation for enabling/disabling Seqera Compute.
+- **Nextflow**
+  - Update `WfManifest` class in Tower to include the new `icon` property introduced in Nextflow.
+  - Add a new migration file to add the new icon column into all the tables of entities that use the `WfManifest` class as @Embedded property.
+- Audit log update: Publish a new *pipeline_updated* audit event whenever a user updates a pipeline.
+- AWS Cloud Provider compute environments: Support running the head job on AWS EC2 instances as an alternative to AWS Batch.
```

Comment:
This one is confusing without additional context, or a link out to docs. Were we tasked with documenting this? This reads as a new compute environment type. And AWS Batch also runs on EC2 instances. @swampie what are we looking at here? A new CE type or just a new head job option inside AWS Batch CEs?